app/preact/src/server/framework-preset-preact.ts

```diff
@@ -3,7 +3,7 @@ import { TransformOptions } from '@babel/core';
 import { Configuration } from 'webpack';
 import { findDistEsm, StorybookConfig } from '@storybook/core-common';
 
-export function babelDefault(config: TransformOptions) {
```

Comment:
If we provide the output type as well, we prevent issues where typescript 'gets confused' and tells you it can't use a non-exported type. `bla is exporting X from 'bla bla' but cannot be named`